libs/sdk-ui-kit/src/ScrollablePanel/ScrollablePanel.tsx

```diff
@@ -27,9 +31,16 @@ const isElementInvisibleCheckDefault: isElementInvisibleType = (
     return false;
 };
 
-// eslint-disable-next-line react/display-name
+const isNotOnInput = (event: React.UIEvent<HTMLDivElement>) => {
+    const target = event.target as HTMLElement;
+    return target.tagName !== "INPUT";
```

Comment:
What if I happen to scroll on an unrelated input?

Reply:
By the truth im not sure why this is here, but is get from previous WidgetScrollPanel component :( Try to find why is it here